CHANGELOG.md

```diff
@@ -4,28 +4,28 @@ All notable changes to this project will be documented in this file.
 
 ## [x.x.x] - UNRELEASED
 
-### Overview
-
-### EDC UI
-
-### EDC-Extensions
-
-#### Major Changes
-
-#### Minor Changes
+### Major Changes
 
-#### Patch Changes
+- Upgrade to core-EDC version `0.1.2`
+- Now using the `Dataspace Protocol`
+- Major changes to the management API
+  - Examples for the new requests are located in the postman collection in the `docs` folder
+  - The `OpenAPI` file has been updated to represent the EDC version `0.1.2`
 
-### Deployment Migration Notes
+### Patch Changes
 
-#### Compatible Versions
+- Fix API Wrapper Broker Server API Spec not showing in client libraries.
 
-- Connector Backend Docker Images:
-  - Dev EDC: `ghcr.io/sovity/edc-dev:{{ VERSION HERE }}`
-  - Sovity EDC CE: `ghcr.io/sovity/edc-ce:{{ VERSION HERE }}`
-  - MDS EDC CE: `ghcr.io/sovity/edc-ce-mds:{{ VERSION HERE }}`
-- Connector UI Docker Image: `ghcr.io/sovity/edc-ui:0.0.1-milestone-8-sovity{{ VERSION HERE }}`
+### Migration Notes
 
+- The `MY_EDC_IDS_BASE_URL` has been renamed to `MY_EDC_PROTOCOL_BASE_URL`
+- The default value of `WEB_HTTP_PROTOCOL_PATH` been changed from `${MY_EDC_BASE_PATH}/api/v1/ids` to `${MY_EDC_BASE_PATH}/api/v1/protocol`
+- The following new environment variables must be set
+  - `EDC_PARTICIPANT_ID`: `provider`
+  - `EDC_JSONLD_HTTPS_ENABLED`: `true`
+  - `EDC_DSP_CALLBACK_ADDRESS`: `http://edc:11003/api/v1/protocol`
+- All endpoints have a `v2` prefix now
+  - example: `http://localhost:11002/api/v1/management/assets/request` is now available at `http://localhost:11002/api/v1/management/v2/assets/request`
```

Comment:
Better: the /api/v1 Management API has been deprecated in favor of the JSON-LD /api/v2 Management API, or something similar, to ensure no hope to that the endpoints are the same